src/population.hpp

```diff
@@ -38,6 +39,19 @@ std::set<std::string> _listChildren(const HighFive::Group& group, const std::set
     return result;
 }
 
+std::set<std::string> _listCustomEnumerations(const HighFive::Group h5Group, const std::set<std::string>& attrs) {
```

Comment:
Is the _list name on purpose even if the function returns a set ?

Reply:
List as in "to list", not referring to the actual datatype. Same goes for `_listChildren`, I guess.

Reply:
Just wanted to check with you.